packages/react/menu/src/Menu.tsx

```diff
@@ -242,15 +239,13 @@ const MenuContentImpl = React.forwardRef((props, forwardedRef) => {
           }, [])}
         >
           <FocusScope
-            // clicking outside may raise a focusout event, which may get trapped.
-            // in cases where outside pointer events are permitted, we stop trapping.
-            // we also make sure we're not trapping once it's been closed
+            // we make sure we're not trapping once it's been closed
             // (closed !== unmounted when animating out)
-            trapped={isPermittedPointerDownOutsideEvent ? false : trapFocus && context.open}
```

Comment:
This bit didn't seem to be needed anymore in my testing, can you recall why we did that?

Reply:
Yeah I'm not sure but it's never ever been fully trapped either way, so I'm not sure why it was ever needed. I'll dig more on the animation use case.

Reply:
Ok I've managed to track down when the change occurred: https://github.com/radix-ui/primitives/pull/394

I've tested the use-case with both `DropdownMenu` and `Popover` and it is definitely not needed anymore.